### ISSUE TITLE
make brightness control exponential

### ISSUE THREAD

**Issue by nova-r** [openSUSE/openSUSEway#112]
As brightness is perceived exponentially and not linearly, it would be more intuitive for brightness control via hotkeys to follow an exponential curve as well. `brightnessctl` supports this already by setting the `-e` flag.

**Comment by FilippoBonazziSUSE** [comment]
Nice proposal. However this has 2 disadvantages:

1. The displayed brightness percentage will then follow what looks to the user like an "irregular" series, i.e. it will go
```
100%
81%
66%
52%
41%
32%
24%
18%
...
```
2. Consequently, it kind of gets weird at low brightness values. Once it goes to low values (e.g. 1 or 0) the number will not change but brightness will.

IMHO we would need a mechanism to make the read-out value exponential as well.

**Comment by nova-r** [comment]
I've added the changes as suggested. It now remaps the exponential scale back to linear for wob
see PR #113

**Comment by FilippoBonazziSUSE** [comment]
Although, on further testing, I have to admit I am not sure I like how this behaves with brightness below e.g. 30% (exponential scaled to linear). Like, once you go below 30% almost nothing changes in brightness, so it feels like we're wasting a piece of the scale at the low end and compressing perhaps meaningful differences at the high end.

**Comment by denisok** [comment]
I agree with @FilippoBonazziSUSE . Currently (with current changes) it works maybe better on high levels of brightness but it degrades when you get lower and also wob doesn't show it right.
